lib/metrics.py

```diff
@@ -192,4 +192,3 @@ def check_metric(**params):
 
   else:
     assert False, "unknown metric"
-
```

Comment:
Please keep an empty line at the end of the file.

Reply:
fixed it